README.md

```diff
@@ -0,0 +1,162 @@
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [To_Do_List] <a name="about-project"></a>
+
+**[To_Do_List]** To-do list is a tool that helps to organize your day. It simply lists the things that you need to do and allows you to mark them as complete. You will build a simple website that allows for doing that, and you will do it using ES6 and Webpack!
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">Html</a></li>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+    <li><a href="https://www.w3schools.com/javascript/">Javascript</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Use webpack to bundle JavaScript.]**
+- **[Learn how to use proper ES6 syntax.]**
+- **[Use ES6 modules to write modular JavaScript.]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+This website has not been deployed. Coming soon...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Install Git on your computer following the steps [here](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- Install node js following the steps [here](https://www.pluralsight.com/guides/getting-started-with-nodejs)
+
+### Setup
+
```

Comment:
- [ ] I kindly suggest that you add the complete steps required to run your project locally because this project requires some commands E.G: `npm install`, `npm run start`, etc.

The below is an example:
### Setup
```

Clone this repository to your desired folder:

````sh`
  cd my-folder
  git clone git@github.com:Kaghenimbale/To-Do-List-app.git
`

### Install

Install this project with:


- Install Node Modules

`   ```sh
`   npm install
`   ```
`
### Usage

To run the project, execute the following command:


- Start the development server

`   ```sh
`   npm start
`   ```
`